BSP/STM32F413/Src/BSP_PLL.c

```diff
@@ -73,25 +74,23 @@ uint32_t BSP_PLL_GetSystemClock(void) {
  * @param   # of microseconds to delay
  * @return  None
 */
-void BSP_PLL_DelayU(uint32_t micro)
-{
-    uint32_t delay = BSP_PLL_GetSystemClock() / 1000000;
-	for(volatile uint32_t i = 0; i < micro; i++)
-	{
-		for(volatile uint32_t j = 0; j < delay; j++);
-	}
+void BSP_PLL_DelayU(uint32_t micro) {
+    uint32_t delay = BSP_PLL_GetSystemClock() / 1e6;
+    delay /= 10; //Through testing, this gets more accurate delays
```

Comment:
can combine lines 78 and 79

Reply:
not combining because delay / 1e7 doesn't make sense. Just having 1 line for regular conversion and another for what we have based on testing seems like an easier way to read the code. Also, this probably gets optimized into one operation either way.